euth/organisations/templates/euth_organisations/organisation_detail.html

```diff
@@ -26,7 +26,7 @@
                     {% if organisation.logo %}
                     <img alt="{{ organisation.name }} logo" src="{{ organisation.logo|thumbnail_url:'organisation_thumbnail' }}">
                     {% else %}
-                    <img alt="opin logo" src="{% static "images/logo.png" %}">
+                    <img alt="opin logo" src="{% static "images/logo.svg" %}">
```

Comment:
In other places we do some detection whether SVG is possible. Should we do it here as well, or is it now safe to assume that all browser support SVG?